2014-01-30	Raphael Amiard  <amiard@adacore.com>

    * cairo-xlib-source: XRenderCreateSolidFill is not available on RHES4, so
      deactivate the path using it.

	  (patch to be removed when upgrading to newer base version of
	  platforms)

	  For M701-036

--- src/cairo-xlib-source.c.orig	2014-01-30 12:20:38.000000000 +0200
+++ src/cairo-xlib-source.c	2014-01-30 12:20:02.000000000 +0200
@@ -456,9 +456,9 @@
     xcolor.blue  = color->blue_short;
     xcolor.alpha = color->alpha_short;
 
-    if (CAIRO_RENDER_HAS_GRADIENTS(dst->display)) {
+    /* if (CAIRO_RENDER_HAS_GRADIENTS(dst->display)) {
 	picture = XRenderCreateSolidFill (dpy, &xcolor);
-    } else {
+    } else */ {
 	XRenderPictureAttributes pa;
 	int mask = 0;
